files.go

```diff
@@ -0,0 +1,159 @@
+package dotfile
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+const (
+	dotfileDir = ".dotfile"
+	dotfile    = "files.json"
+)
+
+type trackedFile struct {
+	Path    string   `json:"path"`
+	Commits []string `json:"commits"`
+}
+
+// Init sets up a file for dotfile to track.
+func Init(path string, fileName string) error {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return fmt.Errorf("%s not found", path)
+	}
+	path, err := filepath.Abs(path)
+	if err != nil {
+		return err
+	}
+
+	if fileName == "" {
+		var err error
```

Comment:
I don't think you need this line, do you?

Reply:
I do because `fileName` is already declared so I can't use the `:=` operator. I agree that it looks bad though. I'll see if there is an easy way to refactor it.